css/contact.css

```diff
@@ -0,0 +1,79 @@
+@media all and (max-width: 500px) {
```

Comment:
## OPTIONAL
### Please note this is not part of this milestone but it would be good to implement this change
- [ ] Kindly remove media query from the mobile version, following the mobile-first approach there should be no media queries, you will add media queries for desktop screen size